### PR TITLE
Issue#79 doc, adds test contract for all LIC5 test-cases

### LINKED ISSUE #79: Add contract for `LIC5` tests

### PR DESCRIPTION
**Pull request by Jonassavas**
Fixes #79 doc, adds test contract for all LIC5 test-cases with pre- and postconditions.

### CODE REVIEW

File: src/test/java/com/decide/DecideTest.java [hunk 1]
```diff
@@ -644,23 +644,39 @@ public void LIC12TestNegativeTwoPointPairs(){
     }
 
     @Test
+    /**
+     * Requirements: See `LIC5` documentation
+     * Contract:
+     *      Precondition: There exists a set of three points with the same
```
Comment (FalkWasTaken) [inline src/test/java/com/decide/DecideTest.java:650]: This precondition is probably inaccurate, specifically the phrasing "there exists a set".

> There exists a set of three points with the same coordinates, or there exists a set of three points with increasing consecutive x-values.

Should probably have something like:
> All sets of three points have the same coordinates or all sets of three points have decreasing x-values.

**Review by FalkWasTaken — CHANGES_REQUESTED** [review]
See my comment for the change suggestions.

File: src/test/java/com/decide/DecideTest.java (hunk 1 above)
Comment (Jonassavas) [reply src/test/java/com/decide/DecideTest.java]: I agree, should be fixed now!

**Review by FalkWasTaken — APPROVED** [review]
Superb!

**Pull request merged**